src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -574,7 +577,7 @@ private static FormulaValue ExactValueType<T>(IRContext irContext, int index, Fo
             }
             else
             {
-                return CommonErrors.RuntimeTypeMismatch(irContext);
+                return CommonErrors.RuntimeTypeMismatch(irContext, null);
```

Comment:
>null [](http://example.com/codeflow?start=67&length=4)

This needs to be localized

Reply:
`StandardErrorHandlingAsync` (the caller) will localize the errors.

Reply:
That sounds inefficient. We're creating an error here, and later we'll create new errors while replacing the message. Why not do the right thing here in the first place?